pysat/tests/classes/cls_instrument_iteration.py

```diff
@@ -0,0 +1,789 @@
+"""Tests for iteration in the pysat Instrument object and methods.
+
+Note
+----
+Base class stored here, but tests inherited by test_instrument.py
+
+"""
+
+import datetime as dt
+import numpy as np
+
+import pandas as pds
+import pytest
+
+import pysat
+from pysat.utils.time import filter_datetime_input
+
+
+class InstIterationTests(object):
+    """Basic tests for `pysat.Instrument` iteration methods.
+
+    Note
+    ----
+    Inherited by classes in test_instrument.py.  Setup and teardown methods are
+    specified there.
+
```

Comment:
Need to add a See Also section here, as done in `..._access.py`.  Do in all similar classes.